__tests__/users.ts

```diff
@@ -557,6 +564,79 @@ describe('query userReadHistory', () => {
   });
 });
 
+describe('query readHistory', () => {
+  const QUERY = `{
+    readHistory {
+      edges {
+        node {
+          timestamp
+          post {
+            id
+            url
+            title
+            image
+            source {
+              image
+            }
+          }
+        }
+      }
+    }
+  }`;
+
+  it('should not authorize when not logged in', () =>
+    testQueryErrorCode(client, { query: QUERY }, 'UNAUTHENTICATED'));
+
+  it("should return user's reading history in descending order", async () => {
+    loggedUser = '1';
+    const createdAtOld = new Date('2020-09-22T07:15:51.247Z');
+    const createdAtNew = new Date('2021-09-22T07:15:51.247Z');
```

Comment:
Maybe also test this one by passing them the other way around? So assigning the new one first then the old?

Reply:
That makes sense. I can probably just switch the first `view` item to have the new date and the second one to have the older date since I think if we create another test, it will serve the same purpose.

Reply:
That makes sense. I can probably just switch the first `view` item to have the new date and the second one to have the older date since I think if we create another test, it will serve the same purpose. What do you think?

Reply:
I'd like to see a test for a consequent page fetch please